### PR TITLE
Suppress more spammy Ratis logs

### PR DESCRIPTION
**Pull request by bf8086**
This change reduces the amount of logs generated on a leader when one of the follower is offline.

### CODE REVIEW

**Comment by bf8086** [comment]
@HelloHorizon @LuQQiu PTAL.

File: conf/log4j.properties [hunk 1]
```diff
@@ -181,6 +181,7 @@ log4j.appender.FUSE_LOGGER.layout.ConversionPattern=%d{ISO8601} %-5p %c{1} - %m%
 log4j.logger.io.grpc.netty.NettyServerHandler=OFF
 
 # Disable noisy INFO logs from ratis
+log4j.logger.org.apache.ratis.grpc.server.GrpcLogAppender=ERROR
```
Comment (HelloHorizon) [inline conf/log4j.properties:184]: Do we want to also ignore the WARN level logs for GrpcLogAppender?

**Review by HelloHorizon — COMMENTED** [review]
@bf8086 Thanks! I leave one minor comment.

File: conf/log4j.properties (hunk 1 above)
Comment (LuQQiu) [inline conf/log4j.properties:184]: Are there useful ERROR log in this class beside the annoying one?

Comment (bf8086) [reply conf/log4j.properties:184]: Yes this line tells log4j to only record `ERROR` level logs and ignore `INFO` or `WARN` level logs.

Comment (bf8086) [reply conf/log4j.properties:184]: This line will still allow log4j to show ERROR level logs for this class. I am sure there are useful WARN and INFO level logs, we can re-enable them per use case when we need to debug certain issues.

Comment (HelloHorizon) [reply conf/log4j.properties:184]: Is there any reason for us to disable the WARN level log for `GrpcLogAppender`? Will it filter out some useful WARN level logs?

Comment (LuQQiu) [reply conf/log4j.properties:184]: that's dangerous.... emmm...

**Review by LuQQiu — APPROVED** [review]
LGTM

File: conf/log4j.properties (hunk 1 above)
Comment (bf8086) [reply conf/log4j.properties:184]: The main purpose is to suppress the excessive messages from `GrpcLogAppender` when a follower is offline:

```
2020-10-14 17:56:30,594 WARN  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-InstallSnapshotResponseHandler: Failed InstallSnapshot: org.apache.ratis.thirdparty.io.grpc.StatusRuntimeException: UNAVAILABLE: io exception
2020-10-14 17:56:30,594 INFO  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-GrpcLogAppender: followerNextIndex = 0 but logStartIndex = 24044765, notify follower to install snapshot-(t:188, i:24044765)
2020-10-14 17:56:30,594 INFO  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-GrpcLogAppender: send FaultToleranceScaleEmbeddedJournal1-masters-1_19200->FaultToleranceScaleEmbeddedJournal1-masters-4_19200#0-t212,notify:(t:188, i:24044765)
2020-10-14 17:56:30,595 WARN  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-InstallSnapshotResponseHandler: Failed InstallSnapshot: org.apache.ratis.thirdparty.io.grpc.StatusRuntimeException: UNAVAILABLE: io exception
2020-10-14 17:56:30,595 INFO  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-GrpcLogAppender: followerNextIndex = 0 but logStartIndex = 24044765, notify follower to install snapshot-(t:188, i:24044765)
2020-10-14 17:56:30,595 INFO  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-GrpcLogAppender: send FaultToleranceScaleEmbeddedJournal1-masters-1_19200->FaultToleranceScaleEmbeddedJournal1-masters-4_19200#0-t212,notify:(t:188, i:24044765)
2020-10-14 17:56:30,595 WARN  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-InstallSnapshotResponseHandler: Failed InstallSnapshot: org.apache.ratis.thirdparty.io.grpc.StatusRuntimeException: UNAVAILABLE: io exception
2020-10-14 17:56:30,595 INFO  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-GrpcLogAppender: followerNextIndex = 0 but logStartIndex = 24044765, notify follower to install snapshot-(t:188, i:24044765)
2020-10-14 17:56:30,595 INFO  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-GrpcLogAppender: send FaultToleranceScaleEmbeddedJournal1-masters-1_19200->FaultToleranceScaleEmbeddedJournal1-masters-4_19200#0-t212,notify:(t:188, i:24044765)
2020-10-14 17:56:30,596 WARN  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-InstallSnapshotResponseHandler: Failed InstallSnapshot: org.apache.ratis.thirdparty.io.grpc.StatusRuntimeException: UNAVAILABLE: io exception
2020-10-14 17:56:30,596 INFO  GrpcLogAppender - FaultToleranceScaleEmbeddedJournal1-masters-1_19200@group-ABB3109A44C1->FaultToleranceScaleEmbeddedJournal1-masters-4_19200-GrpcLogAppender: followerNextIndex = 0 but logStartIndex = 24044765, notify follower to install snapshot-(t:188, i:24044765)
...
```

**Review by HelloHorizon — APPROVED** [review]
LGTM after build pass

**Comment by LuQQiu** [comment]
Test failed in 
```
Error: 8.741 [ERROR] Tests run: 10, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 600.024 s <<< FAILURE! - in alluxio.server.ft.journal.JournalBackupIntegrationTest
Error: 8.741 [ERROR] backupDelegationEmbedded(alluxio.server.ft.journal.JournalBackupIntegrationTest)  Time elapsed: 141.238 s  <<< ERROR!
alluxio.exception.status.UnavailableException: Failed to connect to FileSystemMasterClient @ cf5d2ec710f1:11203 after 1 attempts
	at alluxio.server.ft.journal.JournalBackupIntegrationTest.backupDelegationTest(JournalBackupIntegrationTest.java:477)
	at alluxio.server.ft.journal.JournalBackupIntegrationTest.backupDelegationEmbedded(JournalBackupIntegrationTest.java:354)
```
in second run

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please

**Pull request merged**